test/integration/functional_test.go

```diff
@@ -0,0 +1,42 @@
+// +build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+
+	"k8s.io/minikube/test/integration/util"
+)
+
+func TestFunctional(t *testing.T) {
+	minikubeRunner := util.MinikubeRunner{
+		BinaryPath: *binaryPath,
+		Args:       *args,
+		T:          t}
+	minikubeRunner.EnsureRunning()
+
+	t.Run("DNS", testClusterDNS)
+	t.Run("EnvVars", testClusterEnv)
+	t.Run("Logs", testClusterLogs)
+	t.Run("SSH", testClusterSSH)
+	t.Run("Status", testClusterStatus)
+	t.Run("Addons", testAddons)
+	t.Run("Dashboard", testDashboard)
+	t.Run("ServicesList", testServicesList)
+}
```

Comment:
We weren't deleting before. I kind of like it this way since it's easy to rerun them quickly.